cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -0,0 +1,58 @@
+package workerqueue
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+)
+
+func Create(ws, deploymentID, name string, isDefaultWQueue bool, client astro.Client, out io.Writer) error {
+	var (
+		requestedDeployment    astro.Deployment
+		deloymentToUpdateInput *astro.DeploymentUpdateInput
+		err                    error
+		workerQueueToCreate    astro.WorkerQueue
+		wQueueListToCreate     []astro.WorkerQueue
+	)
+	// TODO get defaults for min-count, max-count and concurrency from API
+
+	// TODO get the deployment
+	requestedDeployment, err = deployment.GetDeployment(ws, deploymentID, client)
+	if err != nil {
+		return err
+	}
+
+	workerQueueToCreate = astro.WorkerQueue{
+		Name:              name,
+		IsDefault:         isDefaultWQueue,
+		MaxWorkerCount:    0,
+		MinWorkerCount:    0,
+		WorkerConcurrency: 0,
+		NodePoolID:        "", // TODO query clusters to get the nodepoolID
+	}
+	// TODO is this the correct way to get worker queues?
+	if len(requestedDeployment.WorkerQueues) > 0 {
+		wQueueListToCreate = append(requestedDeployment.WorkerQueues, workerQueueToCreate) //nolint
+		// TODO if worker-queue exists, update it with requested values
+		// TODO should workerqueue.Create() do this?
+	}
+
+	// no worker-queues exist so create a new list of one worker queue
```

Comment:
wouldn't you also need a way to add the new worker queue to a list of already existing worker queues?